.github/workflows/greenplum-abi-tests.yml

```diff
@@ -102,27 +87,35 @@ jobs:
           fetch-depth: 0 # Specify '0' to fetch all history for all branches and tags.
           path: gpdb_src
 
-      - name: Install abi-dumper
+      - name: Install dependencies and abi-dumper
+        run: |
+          sudo gpdb_src/README.ubuntu.bash
+          sudo apt install -y libuv1-dev libpam0g-dev libldap-dev libipc-run-perl abi-dumper
+          sudo ln -fs python2 /usr/bin/python
+
+      - name: Build libxerces 3.1.2
         run: |
-          yum install -y epel-release
-          yum install -y abi-dumper
-          yum install -y libzstd-static
+          wget https://archive.apache.org/dist/xerces/c/3/sources/xerces-c-3.1.2.tar.gz
```

Comment:
Don't we need our own xerces from Github?

Reply:
As far as I know, we just need specifically version 3.1.2, so it's acceptable to build it here.

Reply:
I could be wrong, could you explain?

Reply:
It's needed for ORCA. It's compiled manually here anyway, so you can probably replace it with our one. The repo is there: <https://github.com/arenadata/gp-xerces>.